src/dir.props

```diff
@@ -7,4 +7,8 @@
     <SolutionFile Condition="'$(OsEnvironment)'=='Windows_NT'">$([System.IO.Path]::Combine($(SolutionRoot),"MSBuild.sln"))</SolutionFile>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(EnableDocumentationFile)' == 'true'">
+    <DocumentationFile>$(OutputPath)$(TargetName).xml</DocumentationFile>
```

Comment:
Is it okay to introduce this $(EnableDocumentationFile) property?

---
In reply to: [78442785](https://github.com/Microsoft/msbuild/pull/1015#discussion_r78442785) [](ancestors = 78442785)